ci/goreleaser/goreleaser.yml

```diff
@@ -122,54 +122,7 @@ dockers:
     dockerfile: ci/images/hybrid/Dockerfile
     extra_files:
       - "ci/images/hybrid/"
```

Comment:
Leave a comment here to indicate that the plugin compiler is built outside goreleaser with a link to the workflow run page.